.github/workflows/helm.yaml

```diff
@@ -0,0 +1,44 @@
+# Copyright 2024 NVIDIA CORPORATION
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Run this workflow on new tags
+name: Helm
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  update-helm-charts:
+    name: Update gh-pages helm repo index
```

Comment:
This also publishes the packages to `gh-pages`, correct?

Reply:
Yes, this action will push to `gh-pages` branch, which will trigger the action there.

Reply:
ah sure, will add

Reply:
done